File: runtime/src/main/java/com/epam/drill/auto/test/agent/penetration/testng/TestNGPenetration.java
package com.epam.drill.auto.test.agent.penetration.testng;

import com.epam.drill.auto.test.agent.penetration.AnnotationStrategy;

public class TestNGPenetration extends AnnotationStrategy {

    public TestNGPenetration() {
        supportedAnnotations.add("@org.testng.annotations.Test");
    }

    @Override
    public String id() {
        return "testng";
    }
}

File: runtime/src/main/kotlin/com/epam/drill/auto/test/agent/penetration/testing/testng/TestNGPenetration.kt
package com.epam.drill.auto.test.agent.penetration.testing.testng

import com.epam.drill.auto.test.agent.penetration.AnnotationStrategy

class TestNGPenetration : AnnotationStrategy() {

    override val id: String = "testng"

    init {
        supportedAnnotations.add("@org.testng.annotations.Test")
    }
}
